scripts/video_generator.py

```diff
@@ -12,12 +12,15 @@ def generate():
     for index, item in enumerate(video_script["timeline"]):
         content = item["content"]
         clip = item["clip"]
+        options = {
+            "engine": item.get("engine", "google")
```

Comment:
how does this work? if your file has engine: eleven wouldn't this fail?
it looks like you're trying to set google as default which we want but this doesn't look right

Reply:
```
item.get("engine", "google")
```
Fetches the `engine` property from the `item` dictionary. It fetches it safely such that if it is not set it will default to `"google"`.

We then store the provided "engine" in an `options` object, which we can pass through to the narration function as required (leaving room for further options, such as which voice to use)